### PR TITLE
Design Static Pages

### PR DESCRIPTION
**Pull request by hazwanizafirah**

### CODE REVIEW

**Comment by hazwanizafirah** [comment]
ready

File: test/controllers/static_pages_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
```
Comment (ngocvt-0484) [inline test/controllers/static_pages_controller_test.rb:3]: dont push test to github. please open gitignore add
```
/test
```
after that
```
git rm --cached -r test
git add .
git commit --amend --no-edit
git push origin your_branch_name -f
```

**Comment by hazwanizafirah** [comment]
Hi. Actually I have a problem where I cannot change to English when I click en. I cannot which one is wrong. Can I know how to fix this?
image

**Comment by hazwanizafirah** [comment]
ready

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (anlvq-1058) [inline app/helpers/static_pages_helper.rb:1]: Delete this file please

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -11,6 +11,12 @@
   </head>
 
   <body>
+    <%= render "layouts/header" %>
+    <div class="container">
+      <% flash.each do |message_type, message| %>
+        <div class="alert alert-<%= message_type %>"><%= message %></div>
+      <% end %>
+
```
Comment (anlvq-1058) [inline app/views/layouts/application.html.erb:19]:
```suggestion
</div>
```

File: app/views/static_pages/help.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<%= provide :title, t(:help) %>
+<html>
+  <head>
+    <title><%= yield(:title) %> | Course Registration System </title>
+  </head>
```
Comment (anlvq-1058) [inline app/views/static_pages/help.html.erb:5]: you don't need full header tag in this file

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<%= provide :title, t(:home) %>
+<html>
+  <head>
```
Comment (anlvq-1058) [inline app/views/static_pages/home.html.erb:3]: you don't need full header tag in this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by hazwanizafirah** [comment]
ready

File: app/helpers/application_helper.rb [hunk 6]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "COURSE REGISTRATION SYSTEM"
```
Comment (huongnt-2545) [inline app/helpers/application_helper.rb:3]:
```suggestion
    base_title = t "h_title"
```

**Comment by hazwanizafirah** [comment]
ready

**Pull request merged**